AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.java

```diff
@@ -179,6 +182,9 @@ public void displayCardQuestion() {
 
     @Override
     protected void displayCardAnswer() {
+        if(mAllFieldsNull && (mCardType != null) && mCardType.equals(getString(R.string.basic_typing_model_name))) {
```

Comment:
just a spacing nitpick
```suggestion
        if (mAllFieldsNull && (mCardType != null) && mCardType.equals(getString(R.string.basic_typing_model_name))) {
```